### PR TITLE
goodbye ei nath

### PR DESCRIPTION
**Pull request by VelardAmakar**
Wand of Severing
Has two modes, Diffindo, which removes arms/hands/legs/feet at high cost, and Sectumsempra, which causes light damage at low cost.

http://gfycat.com/ChiefGlisteningGermanwirehairedpointer

if you give this a controversial tag i don't know what you're doing with yourself
ei nath is a shit spell and needs to go
this is my last pr

### CODE REVIEW

**Comment by Intigracy** [comment]
Why not just make Ei Nath just blow off people's limbs like you're making this wand do?

**Comment by VelardAmakar** [comment]
because a melee semi-disable is terrible
just get rid of it

**Comment by AutismSpider** [comment]
More useless staffs

How about Actual combat spells and not useless tool that get disarm form you 10 secs in

**Comment by VelardAmakar** [comment]
consider getting good and appreciating a nice harry potter reference

**Comment by AutismSpider** [comment]
We don't have any combat spells other than fireball for actually dealing damage.

even then that spell is iffy as shit.

**Comment by VelardAmakar** [comment]
make your own then instead of complaining
you do a lot of complaining

**Comment by AutismSpider** [comment]
Lets be real here you dumb ape

All you done here make a very crappy staff of force.

**Comment by VelardAmakar** [comment]
except you clearly don't know what the staff of force does codewise if you're going to say that

**Comment by AutismSpider** [comment]
Staff has two settings 

low and high power mode

smart people use high power mode to push people into walls for maximum damage

**Comment by VelardAmakar** [comment]
yes but that's not the same as what this does

**Comment by PJB3005** [comment]
IMO this is a good change

**Comment by Iamgoofball** [comment]
Autism spider, stop acting out your name.

Thia is pretty good. :+1:

**Comment by utilitas-** [comment]
I support Autism Spider, this is just a dumb distillation between Ei nath and the staff of force. If you want a Harry Potter reference, make it cast Expecto Patronum, summoning a defensive Patron of the Wizard that blocks some shots. That would be the high setting, and it would be fucking badass. Low setting ought to be Expelliarmus, which just throws away the currently held item.

**Comment by VelardAmakar** [comment]
this is nothing like the fucking staff of force
you don't even code utilitas
stop comparing the two
they do not do close to the same thing
one slams you against walls which can remove limbs
the other is guaranteed to remove limbs harmlessly why do noncoders keep trying to tell coders how code works

**Comment by VelardAmakar** [comment]
honestly if yall want to keep your shitty spell this badly whatever, merge it or don't, i'm done talking about it

even if you don't really like the wand itself it hurts nothing to have variety, and guess what
IT CAN BE REMOVED LATER
the important thing is that this removes ei nath which has zero reason whatsoever to exist for wizards
it's a terrible spell, no other spell or artifact comes close to being as fucking terrible as it
you cannot tweak it to be decent without turning it into a completely different spell because it is fundamentally bad
what is hard to understand about this

anyway
i'm out, i should have just made this pr and ignored it, i really hate dealing with yall cause most of yall act like you never go outside, like bumble over there who thinks that if he whines enough people will care about what he wants

**Comment by utilitas-** [comment]
>it's dumb velard
>BUHH YOU AREN'T A CODER STOP TALKING INTO CODER THINGS
>WHY ARE COMPLAINING WHEN YOU CAN JUST REMOVE IT LATER
Yeah, this is pretty obvious by now

Also, realize this: we do not have to "replace" any spell with any other spell
there is no guideline telling you that there must always be a certain amount of spells or higher
Just fucking remove Ei nath and don't replace it with this dumb shit

**Comment by TheEliminator16** [comment]
Can we merge this already? It's not OP, it removes the hell that is Ei Nath, and it's interesting. It'll also make limb severing more prevalent, maybe we'll see more prosthetic limbs being used.

**Comment by SangriaMonkey** [comment]
Yeah let's just remove every spell but jaunt and have friendly ERP gun wizards every round instead of adding content. Great idea.

**Comment by VelardAmakar** [comment]
okay so i wasn't going to say anything more but
i don't really understand what you're saying there sangria
what
how did you get that from this
i don't understand

**Comment by DrCelt** [comment]
merge now

**Comment by SangriaMonkey** [comment]
I was talking to utilitas there

**Comment by VelardAmakar** [comment]
aaaa i see

**Comment by VelardAmakar** [comment]
yeah the key problems with wizard right now:

1. not enough spells, there needs to be more
2. every spell should not cost the same amount
3. amount of spells you can keep should be increased once there are more spells
4. noclothes should be an upgrade to a spell, not a general ability(or maybe it should be available as an expensive "spell")
5. shields are useless against magic missile, why is this

**Comment by PJB3005** [comment]
@Intigracy is working on more spells IIRC.

**Comment by VelardAmakar** [comment]
@PJB3005 
:ok_hand:

**Comment by Angelite191** [comment]
Velard, I think shield code checks for things it can block individually, and not as families. For example, "Is this a taser round? If so, block it." and not "Is this a projectile? Can I block it? If so, block it" If I'm right, it's an easy but tedious fix.

**Comment by Intigracy** [comment]
I think it's a tad more specific, as in blocking projectile/energy in the case of ablative.

Though I may be wrong.

**Comment by VelardAmakar** [comment]
mm projectiles are coded in a way to where they don't call the needed code to call IsShield() and all that shit
so they lay you out regardless of your shield

**Comment by ComicIronic** [comment]
You don't get hit by spell projectiles, they have an AoE.

**Comment by VelardAmakar** [comment]
i suspected as much, some of that should really be changed

**Comment by Shaska-** [comment]
Here lies the I-Win button.
It has finally had Ei Nath.

Also, you should make it remove a random module (minus battery) from borgs if they get hit, on top of the damage it would otherwise take.
As it is, it's little more than a mosquito bite to them.

File: code/modules/projectiles/projectile/severing.dm [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+					playsound(H.loc, 'sound/weapons/bloodyslice.ogg', 65, 0)
+				else
+					H << "<span class='notice'>You count yourself lucky as the bolt zips past where your [O] should be..</span>"
+			else
+				L.adjustBruteLoss(20)
+				playsound(L.loc, 'sound/weapons/bloodyslice.ogg', 50, 0)
+		else
+			L.adjustBruteLoss(6)
+			playsound(L.loc, 'sound/weapons/bladeslice.ogg', 50, 0)
+	return 1
```
Comment (wild-billy) [inline code/modules/projectiles/projectile/severing.dm]: The above proc could be cleaner:
```dm
/obj/item/projectile/severing/on_hit(var/atom/target,var/blocked)
    if(!isliving(target)) return 0
    var/mob/living/L = target
    if(L.flags & INVULNERABLE) return 0
    if(sever) //Diffindo: severs limbs
        if(prob(23)) return 0 //Diffindo has 23% chance to miss
        if(ishuman(L)) //Limb removal only implemented for humans
            var/mob/living/carbon/human/H = L
            var/datum/organ/external/O = null
            switch(def_zone) //Select target limb, semi-randomly if player isn't aiming at one
                if("head")  O = H.organs_by_name[pick("l_arm","r_arm","l_hand","r_hand")]
                if("chest") O = H.organs_by_name[pick("l_arm","r_arm","l_hand","r_hand","l_leg","r_leg","l_foot","r_foot")]
                if("groin") O = H.organs_by_name[pick("l_leg","r_leg","l_foot","r_foot")]
                else        O = H.get_organ(def_zone) //They're targeting a specific limb, use it
            if(!(O.status & ORGAN_DESTROYED)) O.droplimb(1,1,1) //Slice it off
            else H << "<span class='notice'>You count yourself lucky as the bolt zips past where your [O] should be..</span>"
        else L.adjustBruteLoss(20) //Not a human: just damage them
    else L.adjustBruteLoss(6) //Sectumsempra: plain damage
    playsound(get_turf(L), 'sound/weapons/bladeslice.ogg', 50, 0)
    return 1 
```

**Comment by wild-billy** [comment]
Other than that, looks good code-wise

**Comment by Duny-** [comment]
If you want to remove ei'nath (which should be kept in so we can make fun of scrubs who use it once there are alternatives imo), add damage spells first. The only other direct damage spell kills the wizard more often than the victim, and staves are extremely easily lost thanks to the true I-Win buttons in the game, electrodes, and disarm.

**Comment by d3athrow** [comment]
ban

**Comment by PJB3005** [comment]
@d3athrow good luck getting items WHILE STUNNED kept in hands

**Pull request closed without merging**